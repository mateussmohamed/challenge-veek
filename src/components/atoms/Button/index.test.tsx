import React from 'react'

import { Button } from './index'
import withThemeProvider from '../../theme/withThemeProvider'

describe('[<Button />]', () => {
  const wrap = props => withThemeProvider(() => <Button {...props} />)

  test('matches to snapshot', () => {
    const wrapper = wrap({ primary: true })

    expect(wrapper).toMatchSnapshot()
  })

  describe('when pass secondary prop', () => {
    test('matches to snapshot', () => {
      const wrapper = wrap({ secondary: true })

      expect(wrapper).toMatchSnapshot()
    })
  })
})
