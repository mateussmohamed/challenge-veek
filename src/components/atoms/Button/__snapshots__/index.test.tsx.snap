// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[<Button />] matches to snapshot 1`] = `[Function]`;

exports[`[<Button />] when pass secondary prop matches to snapshot 1`] = `[Function]`;
